docs/sql-migration-guide-upgrade.md

```diff
@@ -23,6 +23,8 @@ license: |
 {:toc}
 
 ## Upgrading From Spark SQL 2.4 to 3.0
+  - Since Spark 3.0, we pad decimal numbers with trailing zeros to the scale of the column for Hive result.
```

Comment:
btw, we should put this description here about hive specific changes? Is it better in `docs/sql-migration-guide-hive-compatibility.md`?

Reply:
It would be great with an example here.

Reply:
Since this is sort of minor, I don't think we should put it in the first place of migration guide.